src/components/DisbursementInviteMessage/index.tsx

```diff
@@ -0,0 +1,156 @@
+          <div className="CardStack__title">Customize invite</div>
+
+          <div className="Note">
+            You can use the organization default message or customized text to
+            invite your receivers to set up a wallet and receive funds for this
+            disbursement. It contains a link to the appropriate wallet at the
+            end of the message. Please check all values for accuracy.
+          </div>
+
+          {isEditMessage ? (
+            <>
+              <div className="ReceiverInviteMessage__options">
+                <RadioButton
+                  id="msg-std"
+                  name="receiver-message"
+                  label={
+                    <Fragment key="msg-std-label">
+                      {"Default message"}{" "}
+                    </Fragment>
+                  }
+                  fieldSize="xs"
+                  value={radioValue.ORGANIZATION}
+                  onChange={handleOptionChange}
+                  checked={selectedOption === radioValue.ORGANIZATION}
+                />
+                <RadioButton
+                  id="msg-cst"
+                  name="receiver-message"
+                  label="Custom message"
+                  fieldSize="xs"
+                  value={radioValue.CUSTOM}
+                  onChange={handleOptionChange}
+                  checked={selectedOption === radioValue.CUSTOM}
+                />
+              </div>
+              {selectedOption === radioValue.CUSTOM ? (
+                <form className="ReceiverInviteMessage__form">
+                  <Textarea
+                    fieldSize="sm"
+                    id="textarea-custom-input"
+                    rows={5}
+                    value={customMessageInput}
+                    onChange={(event) => {
+                      setCustomMessageInput(event.target.value);
+                      updateCustomMessageInput(event);
+                    }}
+                  ></Textarea>
+                </form>
+              ) : (
+                <div className="ReceiverInviteMessage__form">
+                  <Textarea
+                    fieldSize="sm"
+                    id="textarea-standard"
+                    disabled
+                    rows={5}
+                    value={
+                      organization.data.smsRegistrationMessageTemplate ??
+                      standardOrgMessage
+                    }
+                  ></Textarea>
+                </div>
+              )}
+            </>
+          ) : (
+            <div className="ReceiverInviteMessage__form">
+              <Textarea
+                fieldSize="sm"
+                id="textarea-standard"
+                disabled
+                rows={5}
+                value={
+                  draftMessage ??
+                  organization.data.smsRegistrationMessageTemplate ??
+                  standardOrgMessage
+                }
+              ></Textarea>
+            </div>
```

Comment:
Is this the same component as above on line 123 (`id` is the same)? If it is the same component with a different value, we should create a separate component (in this file) to render from a single place to avoid duplicating code.